test/pirls.jl

```diff
@@ -1,25 +1,31 @@
-using DataFrames
-using LinearAlgebra
 using MixedModels
 using Test
 
+using MixedModels: dataset
+
+const fms = Dict(
+    :cbpp => [@formula((incid/hsz) ~ 1 + period + (1|herd))],
+    :contra => [@formula(use ~ 1+age+abs2(age)+urban+livch+(1|urbdist))],
+    :grouseticks => [@formula(ticks ~ 1+year+ch+ (1|index) + (1|brood) + (1|location))],
+    :verbagg => [@formula(r2 ~ 1+anger+gender+btype+situ+(1|subj)+(1|item))],
+)
+
```

Comment:
Will this migrate at some point to the `tests/pls.jl` just in time fitting? For now, this extra layer of indirection doesn't help readability of the tests.

Reply:
I agree that this may not be a good approach here because there are additional arguments needed or used when fitting a GLMM.  Perhaps we could leave this in here for the time being with a mental note to check back and see if there are better ways to organize the dataset/formula/distribution/link/options combinations later.  It may be that reverting to individual formula expressions inside each of the testsets will be the best choice after all.